apps/simple/assemblyai.ts

```diff
@@ -1,14 +1,13 @@
 import { program } from "commander";
-import { AudioTranscriptReader, TranscribeParams } from "llamaindex";
+import { AudioTranscriptReader, TranscribeParams, VectorStoreIndex } from "llamaindex";
```

Comment:
Can you delete this file? We're not using the apps/simple folder anymore

Reply:
Done